### DISCUSSION TITLE
Remove all SOC alerts

### DISCUSSION THREAD

**Discussion by JoshTickles** [Security-Onion-Solutions/securityonion#5593] [Unsupported Versions] [answered]
Hey all, 

Surprisingly, this has never come up for us before today... 

We run a distributed deployment, and we have had the leader assigned to SOC management leave, and no one took over. 
This has caused us to have around 10,000 alerts in the SOC that need to be bulk removed / trashed, or SOC reset. 

I know this was possible in the 16.04 version of SO, but I've not been able to find docs on bulk removal in 2.3. I'm not able to find anything in this discussions area about this either. 

Any advice on how to do so would be appreciated. 

Cheers,

**Comment by dougburks** [comment, marked as answer]
You may be able to put together a script to query Elasticsearch and update the alerts setting `event.acknowledged` to `true`, but this may be more effort than just manually acknowledging the groups of alerts.

Once https://github.com/Security-Onion-Solutions/securityonion/issues/5221 is implemented, then this will be somewhat easier as shown below.

The default view in SOC Alerts groups by `rule.name`, `event.module`, and `event.severity_label`:

![Screen Shot 2021-09-22 at 6 53 45 AM](https://user-images.githubusercontent.com/1659467/134331373-1a69fb3a-42f3-4094-aa21-7c86e87f7913.png)

If you click the `X` on the `rule.name` chip, then you will be left with a very short list which should make it easy to then acknowledge large numbers of alerts quickly:

![Screen Shot 2021-09-22 at 7 44 32 AM](https://user-images.githubusercontent.com/1659467/134337759-d157aac2-06ab-4b68-a7c2-6ebf371f5f58.png)

**Reply by JoshTickles** [reply]
Thanks for the info Doug. 

Looks like #5221 will be our best option in the future, but for now removing `rule.name` in the filters appears to be working.

Also just want to take a second to say thank you to yourself, and your team, for providing such and amazing tool set. You really do make the lives of us dealing with SecOps much easier!

**Reply by dougburks** [reply]
Thanks for your kind words!

To clarify, I don't think removing `rule.name` will actually work correctly until #5221 is implemented. It may look like it's working but if you refresh the Alerts page, I think the acknowledged Alerts will show up again.

**Reply by b4b857f6ee** [reply]
Did you manage to remove all the alert?
